shared-libs/view-map-utils/src/view-map-utils.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+var emit = function() {},
+    viewMapStrings = {},
+    viewMapFns = {},
+    DEFAULT_DDOC_ID = 'default';
```

Comment:
Do we have a DDOC with id "default" or is this just for testing?

Reply:
I've removed the "dummy" ddoc ID and the `defaultViews` value.